modules/core/shared/src/main/scala/gem/Step.scala

```diff
@@ -16,32 +19,140 @@ object Step {
 
   // TODO: better name for this
   sealed trait Base
-  object Base {
+
+  object Base extends BaseOptics {
+
     final case object Bias extends Base
     final case object Dark extends Base
     final case class  Gcal(gcal: GcalConfig) extends Base
     final case class  Science(telescope: TelescopeConfig) extends Base
     final case class  SmartGcal(smartGcalType: SmartGcalType) extends Base
+
+    implicit val EqBias: Eq[Bias.type] =
+      Eq.allEqual
```

Comment:
It seems weird to define `Eq` for `case object`s.

Reply:
I normally would do `Eq.instance` for `Base` in this case

Reply:
Oh I see it was `Eq[Base]` base was defined. Why do you need `Eq[Bias.type]`

Reply:
I did it so that all the cases of `Eq[Base]` could be defined as `a === b`. I thought about `case (Base, Base) => true` instead but went for symmetry all the way through to the arbitraries and cogens.  idk.